azure_ad_admin/icon_azure_ad_admin/triggers/risk_detection/trigger.py

```diff
@@ -0,0 +1,77 @@
+import komand
+from .schema import RiskDetectionInput, RiskDetectionOutput, Input, Output, Component
+
+# Custom imports below
+from icon_azure_ad_admin.util.komand_clean_with_nulls import remove_null_and_clean
+from komand.exceptions import PluginException
+import time
+import requests
+import json
+
+
+class RiskDetection(komand.Trigger):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='risk_detection',
+                description=Component.DESCRIPTION,
+                input=RiskDetectionInput(),
+                output=RiskDetectionOutput())
+        self.risk_level = ""
+        self.found = {}
+
+    def initialize(self):
+        new_risks = self.get_risks()
+
+        try:
+            result = new_risks['value']
+        except KeyError:
+            raise PluginException(cause='Unexpected output format.',
+                                  assistance='The output from Azure Active Directory was not in the expected format. Please contact support for help.',
+                                  data=result)
```

Comment:
```suggestion
                                  data=new_risk.text)
```

Reply:
See my comment above...text isn't available. The function returns the json. It'll bomb out in the function if needed.